### PR TITLE
Merge logging job -- add telemetry_logging role

### PR DESCRIPTION
**Pull request by ayefimov-1**
adding telemetry logging role.

### CODE REVIEW

**Comment by mgirgisf** [comment]
its not failing because of the role , I will create a patch to confirm but I think we need to make this update here also https://github.com/openstack-k8s-operators/telemetry-operator/pull/425/files

File: roles/telemetry_logging/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+Test logging in Openstack
+
+Requirements
+------------
+The following resources are required in the OpenStack cloud
+* network called public
+* network called provate
+* security group called basic with ssh and tcp enabled to the VMs 
+* a flavor called m1.small
+* an image called cirros
```
[the remark this answers is not in the document]
Comment (ayefimov-1) [reply roles/telemetry_logging/README.md]: removed

File: roles/telemetry_logging/tasks/journal_tests.yml
[the remark this answers is not in the document]
Comment (ayefimov-1) [reply roles/telemetry_logging/tasks/journal_tests.yml:1]: ack .... adding main.yml to telemetry_logging role.

**Review by ayefimov-1 — COMMENTED** [review]
Made necessary changes.

File: roles/telemetry_logging/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Dependencies
+------------
+
+Openstack on Openshift deployed. Then logging enabled for Openstack
+
+Example Playbook
+----------------
+
+Each tasks/playbook.yml should be called independently via "ansible.builtin.include_role" with appropriate vars passed:
```
[the remark this answers is not in the document]
Comment (ayefimov-1) [reply roles/telemetry_logging/README.md:26]: Added main.yml for role

**Pull request closed without merging**